tools/testbench/common_test.c

```diff
@@ -56,6 +56,7 @@ int tb_setup(struct sof *sof, struct testbench_prm *tp)
 	sys_comp_module_eq_iir_interface_init();
 	sys_comp_module_multiband_drc_interface_init();
 	sys_comp_module_mux_interface_init();
+	//sys_comp_module_rtnr_interface_init();
```

Comment:
delete if no use.

Reply:
Same, I have a problem with testbench build so I just commented out. Something in this code breaks it.